tools/autograd/gen_python_functions.py

```diff
@@ -172,6 +172,7 @@
     "mul_.Scalar(Tensor(a!) self, Scalar other) -> Tensor(a!)",
     "div.Scalar(Tensor self, Scalar other) -> Tensor",
     "div_.Scalar(Tensor(a!) self, Scalar other) -> Tensor(a!)",
+    "_fused_moving_avg_obs_fq_helper.functional(Tensor self, Tensor observer_on, Tensor fake_quant_on, Tensor running_min, Tensor running_max, Tensor scale, Tensor zero_point, float averaging_const, int quant_min, int quant_max, int ch_axis, bool per_row_fake_quant=False, bool symmetric_quant=False) -> (Tensor running_min, Tensor running_max, Tensor scale, Tensor zero_point, Tensor output, Tensor mask)"  # only used by the functionalization pass # noqa:B950
```

Comment:
We don't want to expose this to python since it's really just used for functionalization.

It also happens to be impossible to expose to python though - the mutable vs. functional versions of this op look identical w.r.t. python (since they have the same base name, args and kwargs. The mutability info isn't part of the python schema information)